authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -786,7 +786,7 @@ private void createRangerServiceIfNecessary(Map<String, String> config, String s
     try {
       rangerClient.getService(serviceName);
     } catch (RangerServiceException rse) {
-      if (rse.getStatus().equals(ClientResponse.Status.NOT_FOUND)) {
+      if (ClientResponse.Status.NOT_FOUND.equals(rse.getStatus())) {
```

Comment:
Does the change here aim to resolve the NEP problem for `res.getSatus().equals`？

Reply:
Yes.